kong/plugins/response-ratelimiting/schema.lua

```diff
@@ -134,7 +136,7 @@ return {
               description =
               "When using the `redis` policy with `redis_ssl` set to `true`, this property specifies if the server SSL certificate is validated. Note that you need to configure the `lua_ssl_trusted_certificate` to specify the CA (or server) certificate used by your Redis server. You may also need to configure `lua_ssl_verify_depth` accordingly.",
               type = "boolean",
-              required = true,
```

Comment:
Doesn't the behavior change now? Before this field was required, but not it's not, right? There are the changed conditional checks, but they don't really enforce policy to be present either?

Reply:
Hm... I wanted to make sure those obsolete fields are not required any more so I changed this `required` to false. However from what I can see those fields also have `default` value defined so I think we can keep them _"required"_ but they'd just take the default value.
So the changeset in this PR should be smaller.

Reply:
To sum it up - I'll revert these two lines